backport/index.ts

```diff
@@ -39,6 +39,7 @@ async function backport() {
   const autoMerge = core.getInput('auto_merge', { required: true }) === 'true';
   const autoMergeMethod = core.getInput('auto_merge_method', { required: true });
   const backportCommandTemplate = core.getInput('manual_backport_command_template', { required: true });
+  const targetPRLabels = core.getInput('target_pr_labels', { required: true }).split(',');
```

Comment:
```suggestion
  const targetPRLabels = core
    .getInput('target_pr_labels', { required: true })
    .split(',')
    .map((label) => label.trim());
```

Could you add that so that it won't break when someone inevitably does `label_1, label_2`?

Reply:
Good call.